src/collective/glossary/browser/views.py

```diff
@@ -98,9 +98,14 @@ def __init__(self, context, request):
 
     def tooltip_is_enabled(self):
         """Check if term tooltip is enabled."""
-        return api.portal.get_registry_record(
+        context = self.context
+        enable_tooltip = api.portal.get_registry_record(
             IGlossarySettings.__identifier__ + '.enable_tooltip'
         )
+        enabled_content_types = api.portal.get_registry_record(
```

Comment:
@rodfersou this condition should not be here; this is redundant with the `is_glossary_object()`